app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -39,6 +41,8 @@ static const int GestureSwipeRight = 1;
 
 static const float kScrollFactor = 20.0f; // Just picked what fell right.
 
+static crow::BrowserWorldPtr sWorld;
```

Comment:
I really don't like having a static shared point especially since we have it now in two places (native-lib.cpp) and this one never gets cleared from what I can tell.